libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -168,7 +173,7 @@ class AC_PrecLand
     AP_Int8                     _retry_behave;      // Action to do when trying a landing retry
     AP_Float                    _sensor_min_alt;     // PrecLand minimum height required for detecting target
     AP_Float                    _sensor_max_alt;     // PrecLand maximum height the sensor can detect target
-    AP_Int8                     _moving;             // True if the landing target is moving (non-stationary)
+    AP_Int16                    _options;            // Bitmask for extra options
```

Comment:
You will have to do a `convert_parameter_width` to go from `AP_Int8` to `AP_Int16`

Reply:
Txs for this.  Making it an Int16 is a good idea.  The parameter has only been in master for about 12hrs though so no need for a parameter conversion.